src/main/java/net/snowflake/client/jdbc/SnowflakePreparedStatementV1.java

```diff
@@ -780,6 +780,11 @@ public void clearBatch() throws SQLException {
     int[] updateCounts = null;
     try {
       if (this.statementMetaData.isArrayBindSupported()) {
+        if (batchSize <= 0) {
+          logger.warn("executeBatch() using array bind with no batch data. Return int[0] directly");
```

Comment:
Use debug here. This is still a valid case.

Reply:
Sure, will do!